### DISCUSSION TITLE
Right to query/access data a service has stored about you

### DISCUSSION THREAD

**Discussion by michielbdejong** [tosdr/tosdr.org#204] [Ideas]
Which services allow you to see an overview of the data they have about you? Maybe we should add that to the checklist, or try to get a list of services that do/don't allow this. This is not the same as "export your data", it would also be about metadata that you didn't explicitly upload to a service (for instance access log)

**Comment by hugoroy** [comment]
↪ Michiel de Jong, janvier 27, 2017 2:44 :
> Which services allow you to see an overview of the data they have about you? Maybe we should add that to the checklist, or try to get a list of services that do/don't allow this. This is not the same as "export your data", it would also be about metadata that you didn't explicitly upload to a service (for instance access log)
> 



This looks like a worthy effort on its own, not necessarily for ToS;DR though.
